packages/react/src/ActionList/Item.tsx

```diff
@@ -294,10 +295,11 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
       'data-inactive': inactive ? true : undefined,
       'data-loading': loading && !inactive ? true : undefined,
       tabIndex: disabled || showInactiveIndicator ? undefined : 0,
-      'aria-labelledby': `${labelId} ${slots.inlineDescription ? inlineDescriptionId : ''}`,
-      'aria-describedby': slots.blockDescription
-        ? [blockDescriptionId, inactiveWarningId].join(' ')
-        : inactiveWarningId,
+      'aria-labelledby': `${labelId} ${slots.trailingVisual ? trailingVisualId : ''}`,
```

Comment:
> How do we decide if trailing visual should be in label or description?

I've added it as a label via `aria-labelledby` instead of a description because if there's text content within trailing visual, it's usually short and connects directly to the main item's text (accessibl, e.g. 

```tsx
<ActionList>
  <ActionList.Item>Tokens Available 
    <ActionList.TrailingVisual>8</ActionList.TrailingVisual>
  </ActionList.Item>
</ActionList>
```
Accessible name would be: "Tokens Available 8". 

I mainly based this on examples I've seen in the wild. If we added it as a description, there's a chance it might be ignored, so I think `aria-labelledby` is a fair bet.

> Are there places in dotcom which would end up with the wrong label / description from this?

I'd say it's very unlikely, since any text within the trailing visual isn't attached to `ActionList.Item`'s accessible name/description currently.

> Re: moving inline description from label to description, can we create a different PR for that because I'm less confident about that change.

Yup that's fair! I think it's worth its own discussion to see if we'd want to switch it or not, as it definitely works both ways, as part of the accessible name, or as part of the description. I'll take that change out of this PR and create another so that we can discuss/see if we want to include it or not!

Reply:
Should now only include the proposed changed for `TrailingVisual`!

Reply:
> I mainly based this on examples I've seen in the wild.

Does this also include within menus/selectpanels?